core/src/main/java/org/infinispan/stats/impl/CacheContainerStatsImpl.java

```diff
@@ -64,7 +64,11 @@ public void resetStatistics() {
       if (getStatisticsEnabled()) {
          for (String cn : cm.getCacheNames()) {
             if (cm.cacheExists(cn)) {
-               cm.getCache(cn).getAdvancedCache().getStats().reset();
+               try {
+                  cm.getCache(cn).getAdvancedCache().getStats().reset();
+               } catch (SecurityException e) {
+                  // no-op, skipping this cache
```

Comment:
I agree with @danberindei: a user has no way of knowing which caches would be included / excluded. I think also that, when security is enabled, only a cache ADMIN would have access to stats ops anyway.